templates/role.tmpl

```diff
@@ -68,8 +160,11 @@
                     <div class="media-content">
                         <div class="content">
                             <p class="title">Are you sure you want to delete this role?</p>
-                            <p>Be careful! Permissions and users use the role id and will have to set back up manually.</p>
-                            <a class="button is-danger" onclick="deleteRole()">Delete role</a>
+                            <p>Be careful! Permissions and users use the role id and will have to set back up
+                                manually.</p>
```

Comment:
Are you trying to say "any permissions and users with this role will also be deleted" - because I think that's a more concise way of saying that (and also the user doesn't really need to know or care that they use the role ID).